src/app/frontend/common/appconfig/appconfig_service.js

```diff
@@ -44,4 +44,12 @@ export class AppConfigService {
       return null;
     }
   }
+
+  getDashboardVersion() {
```

Comment:
It seems to be good place in terms of similar capabilities of all those functions, but you need to double-check if replace won't harm existing functionality. It should not, but in the future we might need more service methods and I would not mix them with your methods (that will be modified by gulp).

So, I'd move it to another service and comment it properly just in case, but it's not required at the moment.